packages/config/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+      "rootDir": "./src",
+      "outDir": "./",
+      "module": "commonjs",
+      "target": "ES5",
+      "declarationDir": "./",
+      "allowJs": true,
```

Comment:
should we consider enforcing [strict](https://www.typescriptlang.org/tsconfig#strict)? Perhaps that can be a followup...

Reply:
Nevermind. I see we are using strict here: https://github.com/RedHatInsights/frontend-components/pull/1892/files#diff-b87edffaa739c304571395ae03f1a69afd153afc0b56374e9c6a7c56e1baa204R12 👍